docs/source/commands/subgraphs.mdx

```diff
@@ -35,7 +35,25 @@ rover subgraph fetch my-graph@my-variant --name accounts
 
 The argument `my-graph@my-variant` in the example above is a [graph ref](../conventions/#graph-refs) that specifies the ID of the Studio graph you're fetching from, along with which [variant](/graphos/graphs/#variants) you're fetching.
 
-**The `--name` option is required.** It specifies _which_ subgraph you're fetching the schema for.
+**The `--name` option is required** It specifies _which_ subgraph you're fetching the schema for.
+
+#### Fetch subgraph schemas from proposals
+
+To fetch a subgraph schema from a [schema proposal](/graphos/delivery/schema-proposals), use the proposal's ID instead of a variant name like so:
+
+```bash
+rover subgraph fetch my-graph@p-101 --name accounts
+```
+
+A proposal's ID is always prefixed with `p-` and followed by a number. You can find the proposal ID in the proposal's URL in [GraphOS Studio](https://studio.apollographql.com?referrer=docs-content). For example, a proposal with the following URL has an ID of `p-101`.
+
+`https://studio.apollographql.com/graph/Example-supergraph/proposal/p-101/home`
+
+<Note>
+
+Though you can use `rover subgraph fetch` to pull subgraph schemas from proposals, you can't use [`rover subgraph publish`](#subgraph-publish) to push schema changes to a proposal.
```

Comment:
Is there another way to push schema changes to a proposal that's worth mentioning?

Reply:
Good point, it's only via Studio which is obvious in the GraphOS docs, but probably not here. Definitely worth adding.